map/mwm_url.hpp

```diff
@@ -40,9 +53,12 @@ class ParsedMapApi
   /// @name Used in settings map viewport after invoking API.
   bool GetViewportRect(m2::RectD & rect) const;
   ApiMarkPoint const * GetSinglePoint() const;
+  vector<RoutePoint> GetRoutePoints() const;
+  // TODO: Need to extend it to bicycle routing
+  bool IsPedestrianRoute() const;
```

Comment:
Да там во фреймворке есть метод, который строку превращает в енам для
роутинга. Лучше его заюзать.
On Sat, Apr 23, 2016 at 4:07 PM Alex Zolotarev <notifications@github.com>
wrote:

> In map/mwm_url.hpp
> <https://github.com/mapsme/omim/pull/2965#discussion_r60828234>:
>
> > @@ -40,9 +53,12 @@ class ParsedMapApi
> >    /// @name Used in settings map viewport after invoking API.
> >    bool GetViewportRect(m2::RectD & rect) const;
> >    ApiMarkPoint const * GetSinglePoint() const;
> > +  vector<RoutePoint> GetRoutePoints() const;
> > +  // TODO: Need to extend it to bicycle routing
> > +  bool IsPedestrianRoute() const;
>
> Лучше сразу енам с двумя значениями.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/mapsme/omim/pull/2965/files/67de44e476cdfcf03e9bbe503dc94a89d300632c#r60828234>
>